Source/WebCore/bindings/js/ScriptModuleLoader.cpp

```diff
@@ -62,10 +63,6 @@
 #include <JavaScriptCore/SourceProvider.h>
 #include <JavaScriptCore/Symbol.h>
 
-#if ENABLE(SERVICE_WORKER)
-#include "ServiceWorkerGlobalScope.h"
```

Comment:
Maybe we can remove WorkerGlobalScope.h here as well.

Reply:
This file does not include that, but does have WorkerOrWorkletGlobalScope.h and WorkletGlobalScope.h so I suspect it needs those. I will create a follow-up fix for the other file though. Seems safest to change that in isolation.